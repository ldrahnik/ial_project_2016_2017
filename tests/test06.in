A 1 B
A 1 C
B 1 D
C 1 D
D 1 E
D 1 F
E 1 G
F 1 G
