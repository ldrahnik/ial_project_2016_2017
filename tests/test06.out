Dijkstra: A -(1)> B -(1)> D -(1)> E -(1)> G
Dijkstra: A -(1)> B -(1)> D -(1)> F -(1)> G
Dijkstra: A -(1)> C -(1)> D -(1)> E -(1)> G
Dijkstra: A -(1)> C -(1)> D -(1)> F -(1)> G
